daemon/src/devices/asteroidosdevice.cpp

```diff
@@ -300,22 +300,19 @@ void AsteroidOSDevice::screenshotReceived(QByteArray data) {
     }
 
     QString fullpath = picturelocation.absolutePath() + "/" + filename;
-    QFile *screenshotFile = new QFile(fullpath);
+    QFile screenshotFile(fullpath);
 
-    qDebug() << fullpath;
+    qDebug() << fullpath << data.size();
 
     QDataStream *dataStream = nullptr;
```

Comment:
Has this been left over?

Reply:
I think file name and size is useful debug message.